site/src/api/api.ts

```diff
@@ -682,12 +682,20 @@ class ApiMethods {
 
 	/**
 	 * @param organization Can be the organization's ID or name
+	 * @param tags to filter provisioner daemons by.
 	 */
 	getProvisionerDaemonsByOrganization = async (
 		organization: string,
+		tags?: Record<string, string>
 	): Promise<TypesGen.ProvisionerDaemon[]> => {
+		const params = new URLSearchParams();
+
+		if (tags) {
+			params.append('tags', encodeURIComponent(JSON.stringify(tags)));
```

Comment:
`URLSearchParams` is already going to encode this for you, which will lead to double encoding and bugs!
```suggestion
			params.append('tags', JSON.stringify(tags));
```